tests/js/spec/views/dashboardsV2/gridLayout/dashboard.spec.tsx

```diff
@@ -196,7 +191,7 @@ describe('Dashboards > Dashboard', () => {
       };
       mount(mockDashboardWithIssueWidget);
       expect(screen.getByText('Test Discover Widget')).toBeInTheDocument();
-      expect(screen.queryByText('Test Issue Widget')).not.toBeInTheDocument();
+      expect(screen.getByText('Test Issue Widget')).toBeInTheDocument();
```

Comment:
This test is called `dashboard does not display issue widgets if the user does not have issue widgets feature flag`. I think we can just remove this test because it's the default behaviour that we expect to see both

I'm fine with throwing that up in a follow up PR if you want this to get in first

Reply:
oops, didn't read the test properly 😅  I went ahead and deleted the test since it doesn't seem to be relevant anymore